Mathlib/Topology/Algebra/Module/Basic.lean

```diff
@@ -309,6 +309,10 @@ instance (priority := 100) continuousSemilinearMapClass [EquivLike F M M₂]
     [s : ContinuousSemilinearEquivClass F σ M M₂] : ContinuousSemilinearMapClass F σ M M₂ :=
   { s with }
 
+instance (priority := 100) HomeomorphClass [EquivLike F M M₂]
```

Comment:
```suggestion
instance (priority := 100) homeomorphClass [EquivLike F M M₂]
```
I don't know this part of the library at all but if you're naming an instance manually it should surely start with a small letter (or maybe even `instHomeomorphClass`?)

Reply:
I just deleted the name and use the generated one. Thank you.